draft-ietf-httpbis-message-signatures.md

```diff
@@ -197,17 +216,19 @@ The following table shows example canonicalized values for header fields, given
 
 |Header Field|Canonicalized Value|
 |--- |--- |
-|`cache-control`|max-age=60, must-revalidate|
-|`date`|Tue, 07 Jun 2014 20:51:35 GMT|
-|`server`|www.example.com|
-|`x-empty-header`||
-|`x-obs-fold-header`|Obsolete line folding.|
-|`x-ows-header`|Leading and trailing whitespace.|
+|`"cache-control"`|max-age=60, must-revalidate|
+|`"date"`|Tue, 07 Jun 2014 20:51:35 GMT|
+|`"server"`|www.example.com|
+|`"x-empty-header"`||
+|`"x-obs-fold-header"`|Obsolete line folding.|
+|`"x-ows-header"`|Leading and trailing whitespace.|
 {: title="Non-normative examples of header field canonicalization."}
 
 ## Dictionary Structured Field Members
 
-An individual member in the value of a Dictionary Structured Field is identified by the lowercased field name, followed by a semicolon `":"`, followed by the member name. An individual member in the value of a Dictionary Structured Field is canonicalized by applying the serialization algorithm described in Section 4.1.2 of {{!StructuredFields=I-D.ietf-httpbis-header-structure}} on a Dictionary containing only that member.
+An individual member in the value of a Dictionary Structured Field is identified by using the parameter `key` on the content identifier for the header. The value of this parameter is a the key being identified, without any parameters present on that key in the original dictionary.
```

Comment:
@jricher as discussed during IETF-109 this feature is still very problematic.

Reply:
I don't necessarily disagree but I wanted to at least make it consistent with the rest of the syntax. An upside is that we don't NEED this functionality in the same way that we used to (with the new `@signature-params` speciality field), and it's also more straightfoward to apply than it was before.